packages/mapviewer/src/router/storeSync/ScrollingModeParamConfig.class.js

```diff
@@ -0,0 +1,28 @@
+import AbstractParamConfig, {
+    STORE_DISPATCHER_ROUTER_PLUGIN,
+} from '@/router/storeSync/abstractParamConfig.class'
+
+export default class ScrollingModeParamConfig extends AbstractParamConfig {
+    constructor() {
+        super({
+            urlParamName: 'ctrl_scroll',
+            mutationsToWatch: ['setScrollWithCtrlOnly'],
+            setValuesInStore: dispatchScrollingModeFromUrlIntoStore,
+            extractValueFromStore: (store) => store.state.ui.scrollWithCtrlOnly,
+            keepInUrlWhenDefault: false,
+            valueType: Boolean,
+        })
+    }
+}
+
+function dispatchScrollingModeFromUrlIntoStore(to, store) {
+    const isEmbed = to.path.includes('embed')
+    const hasScrollParam = 'ctrl_scroll' in to.query
+
+    if (isEmbed && hasScrollParam) {
+        store.dispatch('setScrollWithCtrlOnly', {
+            scrollWithCtrlOnly: true,
+            dispatcher: STORE_DISPATCHER_ROUTER_PLUGIN,
+        })
+    }
```

Comment:
I would use the third argument from `setValuesInStore`, as it gives the url param itself.

```dispatch(to, store, urlParam) 
{
  if (to.path.toLowerCase().includes('embed') { // il me semble que les majuscules fonctionnent aussi dans les routes, à tester.
    store.dispatch('set...', {
      paramName: urlParamValue,
      dispatcher: STORE_DISPATCHER_ROUTER_PLUGIN,
    }
}
}

```